tool_collections/hgv/snpfreq/.shed.yml

```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/snpfreq
-owner: devteam
+categories:
+- Variant Analysis
```

Comment:
@nsoranzo I'm not sure a user that is searching for statistic tools expects this tool. As well as the bam-stat tools. Imho these should not be under statistics.